.github/workflows/refresh_dist.yml

```diff
@@ -49,13 +52,15 @@ jobs:
         with:
           commit_message: Refresh dist files
 
-      - name: Remove label
-        uses: actions/github-script@v6
+      # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#using-data-from-the-triggering-workflow
+      - name: Save PR URL
+        env:
+          PR_URL: https://github.com/${{ github.repository }}/pull/${{ github.event.number }}
+        run: |
+          mkdir -p ./pr
+          echo $PR_URL > ./pr/pr_url
```

Comment:
Could we use https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable here instead of an artifact?

Reply:
Apparently not. This ultimately borrowed heavily from the [GitHub docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#using-data-from-the-triggering-workflow). It's really strange that this is the official method.

That said, I did spend a bit of time trying to figure out if there was a less indirect way to get the PR number from the triggering workflow. [This comment](https://github.com/orgs/community/discussions/25220?sort=new#discussioncomment-8697399) was as close as I could find (from a long discussion about this curious omission in the API), but I wasn't able to get it working myself. Maybe you'd like to take a crack at it?

Reply:
Ah, the data needs to cross workflows. Never mind then.